flowsa/common.py

```diff
@@ -43,7 +43,7 @@
 fbsoutputpath = outputpath + 'FlowBySector/'
 
 paths = Paths
-paths.local_path = os.path.realpath(paths.local_path + "/flowsa")
+paths_local_path = os.path.realpath(paths.local_path + "/flowsa")
```

Comment:
I believe local_path is a variable within the class Paths, so this needs to remain paths.local_path (I think)

Reply:
yes, my mistake, I changed it back